content/definitions/czym-sa-dumb-smart-komponenty.md

```diff
@@ -0,0 +1,34 @@
+---
+question: Czym są dumb i smart komponenty?
+---
+
+`Smart komponenty` wykonują logiczne operacje, strzelają po dane, wykonują obliczenia, modyfikują dane, które później mogą
+przekazać do innych komponentów. Często `smart komponenty` nazywamy kontenerami, których zadaniem jest właśnie przejęcie całego mechanizmu strzelania po API, czy
+operacji na danych.

```

Comment:
"Smart komponenty wykonują logiczne operacje. Mogą strzelać po dane i je modyfikować by przesłać je w odpowiedniej formie do innych, dumb komponentów. " ? 

This part: 
"...których zadaniem jest właśnie przejęcie całego mechanizmu strzelania po API, czy operacji na danych." - is rather a duplication of content from first line, don't u think? 

Ok, let's try to wrap it up. 

"`Smart komponenty` wykonują logiczne operacje, strzelają po dane, modyfikują je i przesyłają przygotowane wartości do dumb komponentów. Nie odpowiadają za wyświetlanie informacji i nie zawierają styli CSS. 
Często `smart komponenty` nazywamy kontenerami.

`Dumb komponenty` natomiast są to komponenty, których rolą jest wyłącznie wyświetlenie otrzymanych danych. Zawierają HTML i CSS. Nie używają zewnętrznych zależności (bibliotek), za wyjątkiem tych odpowiedzialnych za prezentacje informacji"

🤔 🤔 🤔 🤔 🤔 ❓ ❔ ❓ ❔ ❓ ❔

Reply:
MMm you are like Redaktor Naczelny.  Good idea.